std/experimental/ndslice/slice.d

```diff
@@ -2934,7 +2941,7 @@ unittest
 
 private enum isSlicePointer(T) = isPointer!T || is(T : PtrShell!R, R);
 
-private struct LikePtr {}
+package struct LikePtr {}
```

Comment:
@klickverbot You can use random access range interface and `sliced` function. `@LikePtr` is for optimisation reasons. It may be exposed in the future, but I need to write a proper documentation of how it should be used first. The number of use cases for `@LikePtr` is very small, and all of them seems should be a part of the package.